cmd/run.php

```diff
@@ -22,8 +22,8 @@
 
 $breweryScrapper->collectData();
 $breweriesData = $breweryScrapper->getData();
-$providers = $providerService->getProviders($breweriesData);
-$test = $breweryFactory->create($breweriesData);
+$breweriesFactory = $breweryFactory->create($breweriesData);
```

Comment:
I would like to rename `breweriesFactory` to `breweries` or `breweriesCollection` due to this variable keep Collection object of Brewery objects. It is not BreweryFactory class